internal/miner/miner.go

```diff
@@ -468,8 +468,7 @@ func calculateInterlink(
 	origDifficulty DifficultyMetrics,
 	currentInterlink [][]byte,
 ) [][]byte {
-	interlink := make([][]byte, len(currentInterlink))
-	copy(interlink, currentInterlink)
+	interlink := currentInterlink
```

Comment:
The purpose of this was to avoid modifying the original interlink value by copying instead of reusing its slice. However, in practice it shouldn't actually matter since we'll never use the previous interlink value again

Reply:
Right, I understood why it was done the way it was done. The golangci-lint complaint was that we're later appending to a slice which was created as non-zero. I'm not sure exactly why that would be such a big thing, but this was a simple enough change and while it does change how it works, in practice the results are the same, as you said.